modules/integration/src/test/java/org/apache/fluo/integration/TestTransaction.java

```diff
@@ -210,27 +195,6 @@ public ScannerBuilder scanner() {
   }
 
   @Override
-  public Map<RowColumn, String> gets(Collection<RowColumn> rowColumns) {
```

Comment:
It's nice how the default methods simplify the different implementations of Transaction